provider/gateway/rest/client.go

```diff
@@ -263,6 +265,55 @@ func (c *client) Status(ctx context.Context) (*provider.Status, error) {
 	return &obj, nil
 }
 
+func (c *client) Validate(ctx context.Context, gspec dtypes.GroupSpec) (provider.Validate, error) {
+	uri, err := makeURI(c.host, validatePath())
+	if err != nil {
+		return provider.Validate{}, err
+	}
+
+	if err = gspec.ValidateBasic(); err != nil {
+		return provider.Validate{}, err
+	}
+
+	var obj provider.Validate
```

Comment:
Can this be declared down below, where it is used?

Reply:
💯